src/wallet/rpcdump.cpp

```diff
@@ -475,9 +475,7 @@ UniValue importwallet(const UniValue& params, bool fHelp)
             zerocoinEntry.nHeight = stoi(vstr[6]);
             zerocoinEntry.id = stoi(vstr[7]);
             if(vstr.size()>8){
-                uint256 ecdsaSecretKey;
-                ecdsaSecretKey.SetHex(vstr[8]);
-                zerocoinEntry.ecdsaSecretKey = std::vector<unsigned char>(ecdsaSecretKey.begin(),ecdsaSecretKey.end());
+                zerocoinEntry.ecdsaSecretKey = std::vector<unsigned char>(vstr[8].begin(),vstr[8].end());
```

Comment:
This is not right. `vstr[8]` is a hex string, not binary. We need to convert it to binary first. We can use `ParseHex` function to convert hex string to binary.